examples/IRMQTTServer/IRMQTTServer.h

```diff
@@ -78,11 +78,11 @@ const bool kInvertTxOutput = false;
 
 // Default GPIO the IR demodulator is connected to/controlled by. GPIO 14 = D5.
 // Note: GPIO 16 won't work on the ESP8266 as it does not have interrupts.
-const int8_t kDefaultIrRx = -1;  // <=- CHANGE_ME (optional)
+const int8_t kDefaultIrRx = 21;  // <=- CHANGE_ME (optional)
```

Comment:
why 21?

Reply:
if this is the RX pin, it is better to add a define on thmedia.h and use it here, so it is more readble

Reply:
Yes, 21 is the RX pin for IR signals. I made this change adding `thmediaIrRx` on thmedia.h